src/containers/ReportArticle/ReportArticle.test.js

```diff
@@ -0,0 +1,86 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import { ReportArticle } from './index';
+import { fetchTypesResponse, user } from './state/mock';
+import { article } from '../../utils/testHelpers';
+import { REPORT_WITHOUT_DATA } from "./state/types";
+
+const props = {
+  reports: {
+    errors: "",
+    hasError: false,
+    hasErrors: true,
+    isReporting: false,
+    message: "",
+    reportTypes: fetchTypesResponse.data,
+    success: false,
+  },
+  articleReporting: {
+    success: false,
+  },
+  reportArticle: jest.fn(),
+  fetchViolationTypes: jest.fn(),
+  article: {
+    ...article,
+    author: user,
+  },
+};
+const historyMock = {
+  push: jest.fn(),
+};
+
+
+const wrapper = mount(<ReportArticle {...props} reports={props.reports} />);
+const instance = wrapper.instance();
+
+
+describe('<ReportArticle container', () => {
```

Comment:
Refactor this test suite title

Reply:
I have refactored it to `The Report Article container` as advised. Thanks